modules/microshift-get-cluster-id-kubesystem.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// microshift_support/microshift-getting-cluster-id.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-get-cluster-id-kubesystem_{context}"]
+= Getting your cluster ID using kube-sytem namespace
+
+.Procedure
+
+* Get the namespace UID of the running cluster by entering the following command:
++
+[source,terminal]
+----
+$ # oc get namespaces kube-system -o jsonpath={.metadata.uid}
+----
+
+* Get the namespace UID of a non-running cluster by entering the following command:
++
+[source,terminal]
+----
+# cat /var/lib/microshift/cluster-id
+----
+
+[NOTE]
+====
+The cluster-id file contains the namespace UID for a non-running cluster. The cluster-id file is found in the directory '/var/lib/microshift'.
```

Comment:
```suggestion
The MicroShift service copies the `kube-system` namespace metadata UID to the `cluster-id` file so that the UID can be accessed when the cluster is not running. The `cluster-id` file is found in the `/var/lib/microshift` directory.
```

Reply:
done